docker-entrypoint.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/sh
 if ! which -- "${1}"; then
   # first arg is not an executable
+  rm /tmp/.X99-lock -f > /dev/null || ``
```

Comment:
why does this need the `|| ``` part. seems like whether this file exits or not it will be deleted with no messaage.

Reply:
Thats like a Try-Catch :D When the file doesnt exist and i dont have the 2 pipes in it the container wont start.

Reply:
i updated the docker-entrypoint.sh

now the deletion is only executed when the file exists.

Reply:
I like this better, could we make it a single line, like
`if [ -e /tmp/.X99-lock ]; then rm /tmp/.X99-lock -f; fi`